src/playercards/cards/NkosiMabati3.ttslua

```diff
@@ -59,11 +69,35 @@ function makeXMLButton()
   )
 end
 
+-- Create XML button to prompt choosing a sigil; acts as this card's helper
+function initialize()
+  self.UI.setXmlTable({
```

Comment:
There should be a condition: If there's already a sigil defined (e.g. because it was saved and loaded), this should skip the "choose sigil" XML